edge/pkg/eventbus/common/util/common.go

```diff
@@ -66,7 +66,7 @@ func HubClientInit(server, clientID, username, password string) *MQTT.ClientOpti
 
 		caCert, err := ioutil.ReadFile(eventconfig.Config.TLS.TLSMqttCAFile)
 		if err != nil {
-			klog.Errorf("Failed to read TLSMqttCAFile")
+			klog.Errorf("Failed to read TLSMqttCAFile: %v", err)
```

Comment:
Let's use lowercase for all the log, ref: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
ok，modificaion is complete